test/helpers.go

```diff
@@ -88,19 +88,18 @@ func RandomReader(seed, size int) *bytes.Reader {
 
 // SetupTarTestFixture extracts the tarFile to outputDir.
 func SetupTarTestFixture(t testing.TB, outputDir, tarFile string) {
-	err := System("sh", "-c", `(cd "$1" && tar xzf - ) < "$2"`,
-		"sh", outputDir, tarFile)
+	f, err := os.Open(tarFile)
+	defer f.Close()
 	OK(t, err)
-}
 
-// System runs the command and returns the exit code. Output is passed on to
-// stdout/stderr.
-func System(command string, args ...string) error {
-	cmd := exec.Command(command, args...)
+	cmd := exec.Command("tar", "xzf", "-")
```

Comment:
... and here it is, so it can be included in helpers.go

```Go
// resetReadOnlyFlagAll resets windows read-only flag
// set on path and any children it contains.
// The flag is set by git and has to be removed.
// os.Remove refuses to remove files with read-only flag set.
func resetReadOnlyFlagAll(t testing.TB, path string) {
	fi, err := os.Stat(path)
	if err != nil {
		t.Fatalf("resetReadOnlyFlagAll(%q) failed: %v", path, err)
	}
	if !fi.IsDir() {
		err := os.Chmod(path, 0666)
		if err != nil {
			t.Fatalf("resetReadOnlyFlagAll(%q) failed: %v", path, err)
		}
	}
	fd, err := os.Open(path)
	if err != nil {
		t.Fatalf("resetReadOnlyFlagAll(%q) failed: %v", path, err)
	}
	defer fd.Close()
	names, _ := fd.Readdirnames(-1)
	for _, name := range names {
		resetReadOnlyFlagAll(t, path+string(filepath.Separator)+name)
	}
}
```

If I insert `resetReadOnlyFlagAll(t, tempdir)` right after `	SetupTarTestFixture(t, tempdir, repoFixture)` it helps a lot.